app/models/tag_substitution/substitution.rb

```diff
@@ -19,7 +19,7 @@ class TagSubstitution::Substitution
   delegate :disable_match_expectation, to: :tag_substituter, allow_nil: true
 
   delegate :friendly_name, to: :sample, prefix: true
-  validates :sample_id, :library_id, presence: true
+  validates :sample_id, presence: true
```

Comment:
We don't generate a library id for the long read manifests, as they aren't libraries. But we *do* update the aliquot attributes, like library-type (to indicate what library will be made). This triggers the tag substitution mechanism on re-upload.

This removes the need for library id to be provided, but will still match against it. (So we'll only be able to update the attributes for non-library aliquots)